suitest.js

```diff
@@ -49,36 +49,38 @@ const {NETWORK_PROP, NETWORK_METHOD} = require('./lib/constants/networkRequest')
 const HAD_NO_ERROR = require('./lib/constants/hadNoError');
 
 // Network
-const {webSocketsFactory} = require('./lib/api/webSockets');
-const {unusedExpressionWatchersFactory} = require('./lib/utils/unusedExpressionWatchers');
+const webSocketsFactory = require('./lib/api/webSockets');
+const unusedExpressionWatchersFactory = require('./lib/utils/unusedExpressionWatchers');
 const ipcClient = require('./lib/testLauncher/ipc/client');
 const ipcServer = require('./lib/testLauncher/ipc/server');
 
 // Contexts
 const Context = require('./lib/utils/Context');
 const AuthContext = require('./lib/utils/AuthContext');
 const {configFactory} = require('./config');
-
+const {setUpRaven} = require('./lib/utils/sentry/Raven');
 const {warnLauncherAndLibHasDiffVersions} = require('./lib/utils/packageMetadataHelper');
 const {createLogger} = require('./lib/utils/logger');
 
 // Publicly available API goes here
-class SUITEST_API {
-	constructor(exitOnError, instanceDependencies) {
-		// default instance dependencies
+class Suitest {
```

Comment:
I think that we need use SuitestApi, or SuitestAPI beacause "suitest" library already exists and contains Suitest class

Reply:
in docs we have it like `suitest` but for me it does not meter, @andriichumak what class name should we use?

Reply:
I think Suitest is fine. I think @JeKachan refers to suitest test runner that some guy created long ago and already abandoned.